### PR TITLE
Add unitest

### PR DESCRIPTION
**Pull request by GustavoZHA**

### CODE REVIEW

File: src/test/java/org/fundacionjala/at13/spaceinvaders/bulletTest.java [hunk 1]
```diff
@@ -261,4 +302,7 @@ public void itShouldReturnBulletToPosition0X0YWhenItGetYLimitForMovingBulletUp()
         bulletAlien.movingBulletUp();
         assertEquals(0, bulletAlien.getPositionY());
     }
+
+
+
```
Comment (jpgonzales27) [inline src/test/java/org/fundacionjala/at13/spaceinvaders/bulletTest.java:307]: these spaces should be removed in the code

Comment (danielaSantaCruz) [inline src/test/java/org/fundacionjala/at13/spaceinvaders/bulletTest.java:307]: these blank lines should be deteled

Comment (GustavoZHA) [reply src/test/java/org/fundacionjala/at13/spaceinvaders/bulletTest.java:307]: ok

Comment (GustavoZHA) [reply src/test/java/org/fundacionjala/at13/spaceinvaders/bulletTest.java:307]: That's right

**Review by jpgonzales27 — APPROVED** [review]

**Review by danielaSantaCruz — APPROVED** [review]

**Review by cristianfchq — APPROVED** [review]

**Pull request merged**